helm/happa-chart/templates/network-policy.yaml

```diff
@@ -0,0 +1,20 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  namespace: {{ .Values.namespace }}
+  name: {{ .Values.name }}-network-policy
+spec:
+  podSelector:
+    matchLabels:
+      app: {{ .Values.name }}
+  ingress:
+  - ports:
+    - port: {{ .Values.port }}
+      protocol: {{ .Values.protocol }}
+  egress:
+  - to:
```

Comment:
Let's keep one format everywhere. afaik we dropped just `- {}` previously to allow all traffic